eng/merge-asset-manifests.proj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks\Microsoft.DotNet.UnifiedBuild.Tasks.csproj" />
+    <ProjectReference Include="$(RepoProjectsDir)$(RootRepo).proj" />
```

Comment:
I think this also needs metadata to not actually build the project:

```suggestion
    <ProjectReference Include="$(RepoProjectsDir)$(RootRepo).proj" BuildReference="false" />
```

Reply:
Let's try to add this as well, thank you!